README.md

```diff
@@ -1,9 +1,22 @@
-# JInput
-
 [![Maven Central](https://img.shields.io/maven-central/v/net.java.jinput/coreapi.svg)](https://maven-badges.herokuapp.com/maven-central/net.java.jinput/coreapi)
 [![Javadocs](http://www.javadoc.io/badge/net.java.jinput/coreapi.svg)](http://www.javadoc.io/doc/net.java.jinput/coreapi)
 
-Library for access to input devices.
+JInput is a cross-platform Java API designed for the discovery and polling of various input devices, encompassing
+keyboards, mice, joysticks, and gamepads.
+
+The supported range of devices is contingent on the underlying operating system. JInput utilizes
```

Comment:
This was copied from the wiki. Let me know if anything is incorrect and I can update it 👍